diesel/src/sqlite/mod.rs

```diff
@@ -13,3 +13,21 @@ pub mod query_builder;
 pub use self::backend::{Sqlite, SqliteType};
 pub use self::connection::SqliteConnection;
 pub use self::query_builder::SqliteQueryBuilder;
+
+/// Trait for the implementation of a SQLite aggregate function
+///
+/// This trait is to be used in conjunction with the `sql_function!`
+/// macro for defining a custom SQLite aggregate function. See
+/// the documentation [there](../../diesel_derives/macro.sql_function_proc.html) for details.
```

Comment:
Here is the link I mentioned in Gitter by the way

Reply:
I think 
```suggestion
/// the documentation [there](../prelude/macro.sql_function.html) for details.
```

should also link to the corresponding documentation, but in such a way that it will not break on docs.rs

Reply:
I don't think it is working with
```
cargo doc --no-deps --features "sqlite extras"
```

Reply:
This is what is in `target/doc/diesel/prelude`

![image](https://user-images.githubusercontent.com/15683724/78302218-fb4f9280-74ff-11ea-99ac-aa9472f4f0ad.png)

Reply:
Try building with a newer rustdoc version :wink:
`cargo +1.42.0 doc --features "sqlite"` leads to different results, including a `macro.sql_function.html` file.
(We will bump to rustdoc version before the next release to make this work)